Documentation/FirstProject/CreateRootPage.rst

```diff
@@ -14,55 +14,114 @@ It serves as the top-level page in the page tree and is essential for
 you will learn how to create a root page in TYPO3 v13.
 
-..  note::
+..  admonition:: Prerequisites
+
     *   You have TYPO3 :ref:`installed <t3start:installation-ddev-tutorial>`
         and can log into the backend.
     *   You have access to the Page module and permission to create pages.
-    *   You are familiar with the basic layout of the TYPO3 backend.
+    *   You are familiar with the basic layout of the TYPO3
+        `Backend <https://docs.typo3.org/permalink/t3start:backend>`_.
+
+..  contents:: Steps to create a root package
+
+..  _create-root-page-new-page:
+
+Create a new page on root level
+===============================
+
+Once you log into the TYPO3 backend, locate the :guilabel:`Web > Page` module on the left-hand
+side of the screen. Click on the Page module to open the page tree.
+
+..  figure:: /Images/ManualScreenshots/CreateRootPage/CreateNewPage.png
+    :alt: The "Page" module in the TYPO3 backend with a context menu to create a new page
+
+*   In the page tree, right-click on the "root level" or the top node of the page
+    tree (if no pages exist yet, this will likely be labeled as "site" or similar).
+    A context menu will appear.
+
+*   From the context menu, select "New". This will open a form for creating a new
+    page.
+
+..  _create-root-page-configure:
+
+Configure the newly created page
+================================
+
+Now, you need to configure the new page:
+
+*   In the form that appears, give your new page a name.
+    This will be the label of your root page in the page tree.
+
+*   Under the :guilabel:`Behaviour` tab, look for the checkbox called
+    "Use as Root Page". Ensure this is checked.
+
+..  figure:: /Images/ManualScreenshots/CreateRootPage/SetRootPage.png
+    :alt: Page properties, tab "Behaviour", highlighting the "Use as root page"
+
+..  _create-root-page-save:
+
+Save the page
+=============
+
+Once you have filled in the necessary information and selected the
+"Use as Root Page" option, click the "Save" button at the top of the
+form. Your new root page will now appear in the page tree on the left.
+The page is only visible in the Backend at this point
+
+..  figure:: /Images/ManualScreenshots/CreateRootPage/PageCreated.png
+    :alt: Backend module "Page" with a newly created but still disabled root page
+
+    The root page is disabled, therefore it is marked with the red icon.
+    You can enable it in the context menu.
+
+..  _create-root-page-site:
 
-..  rst-class:: bignums-xxl
-#.  Access the Page module
+Configure the site
+==================
 
-    Once you log into the TYPO3 backend, locate the :guilabel:`Web > Page` module on the left-hand
-    side of the screen. Click on the Page module to open the page tree.
+You need a `Site configuration <https://docs.typo3.org/permalink/t3start:site-management>`_.
+If you started in a freshly installed TYPO3 installation the site configuration
+was auto-created and you can edit it:
```

Comment:
```suggestion
All sites must have a `site configuration <https://docs.typo3.org/permalink/t3start:site-management>`_.
If you have created a new page in the root of the page tree, the site configuration has been
created automatically and you can edit it:
```

This happens for all pages that are created on root level, independently of whether you are in a fresh installation or not. It even happens if you don't select "Use as root page" in the "Behavior" tab.